app/helpers/samples_helper.rb

```diff
@@ -611,4 +627,7 @@ def filter_by_sample_ids(samples, requested_sample_ids)
     requested_sample_ids.class.name
     samples.where(id: requested_sample_ids.is_a?(Array) ? requested_sample_ids : JSON.parse(requested_sample_ids))
   end
+
+  # Callable without include/extend
+  module_function :samples_by_metadata_field
```

Comment:
Why?

Reply:
this just let me call it as a class method `SamplesHelper.samples_by_metadata_field` without having to pull in all the other methods since it's defined as an instance method?

Reply:
How hard would it be to just make them static?

Reply:
changed to static 👍